cql2pgjson/src/test/java/org/z3950/zing/cql/cql2pgjson/ForeignKeyGenerationIT.java

```diff
@@ -67,17 +73,24 @@ public void foreignKeyFilter1() throws Exception {
     assertThat(cql("id == tableb.tableaId"), containsInAnyOrder("test1", "test2", "test3"));
   }
 
+  @Test
+  public void foreignKeyFilterParentChild() throws Exception {
+    assertThat(cql("id ==  tabled.tabledId"), containsInAnyOrder("test0", "test1","test2", "test3"));
```

Comment:
`tabled.tabledId` does not exists. What does this intend to test?

Reply:
this is intended to test the filter functionality, though I do agree that it looks weird.  This may need to look like this instead but it fails my unit tests:
tablea.tableaId == id.  But I need to think about it more

Reply:
This unit test is no longer valid.  It will need to be removed in a future update